libs/sdk-ui-vis-commons/src/legend/PopUpLegend/RowLegend.tsx

```diff
@@ -42,13 +42,16 @@ export const RowLegendIcoButton: React.FC<IRowLegendIcoButton> = (props) => {
         return null;
     }
 
-    const handleOnClick = () => {
+    const handleOnClick = (e: React.MouseEvent<HTMLDivElement>) => {
+        e.preventDefault();
+        e.stopPropagation();
```

Comment:
Why is this change necessary?

Reply:
That was bug in vis-common ... legends fire after click widgetSelect by clicking on legend ico